src/backend/partaj/core/models/referral_urgencylevel_history.py

```diff
@@ -0,0 +1,60 @@
+"""
+R Referral historic urgency model in our core app.
+"""
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ReferralUrgencyLevelHistory(models.Model):
+    """
+    Referral historic urgency model.
+    Instances of this model should only be created by administrators.
+    """
+
+    id = models.UUIDField(
+        verbose_name=_("id"),
+        help_text=_("Primary key for the referral urgency level history as UUID"),
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False,
+    )
+
+    referral = models.ForeignKey(
+        verbose_name=_("referral"),
+        help_text=_("Referral to which urgency level history"),
+        to="Referral",
+        on_delete=models.CASCADE,
+        related_name="urgencylevel_history",
+    )
+
+    old_ReferralUrgency = models.ForeignKey(
+        verbose_name=_("urgency"),
+        help_text=_("Old urgency level"),
+        to="ReferralUrgency",
+        on_delete=models.PROTECT,
+        related_name="+",
+    )
+
+    new_ReferralUrgency = models.ForeignKey(
+        verbose_name=_("urgency"),
+        help_text=_("new urgency level"),
+        to="ReferralUrgency",
+        on_delete=models.PROTECT,
+        related_name="+",
+    )
+
+    new_Referralurgency_explanation = models.TextField(
```

Comment:
We're already on an urgency change history object. We can just call the field "explanation" as the we know what it's an explanation for.

```suggestion
    explanation = models.TextField(
```